Extensions/lethe.js

```diff
@@ -1,7 +1,7 @@
-//* TITLE Lethe **//
-//* VERSION 1.1.1 **//
-//* DESCRIPTION Forgets posts once they scroll off the screen. **//
-//* DEVELOPER hobinjk **//
+//* TITLE Hermes **//
+//* VERSION 1.2.0 **//
+//* DESCRIPTION Helps speed up your Tumblr experience **//
```

Comment:
description is more vague then previous description :/

are we aiming to make lethe more of a "performance tweaks" extension? Are there other things we might want to migrate here eventually? (not in this PR, but soon)

would also appreciate a extension name change (cf. old stats, colorquotes) with notification.